internal/testing/e2e/common.go

```diff
@@ -162,15 +162,15 @@ func (om *om) MMFConfigHTTP() *pb.FunctionConfig {
 // that parsing bugs can't hide logic bugs.
 const registrationInterval = time.Millisecond * 200
 const proposalCollectionInterval = time.Millisecond * 200
-const pendingReleaseTimeout = time.Millisecond * 200
+const pendingReleaseTimeout = time.Second * 1
```

Comment:
why change these from 200ms to 1s?

Reply:
This was introduced in #1310 and I needed this change too.